.github/workflows/codecov.yaml

```diff
@@ -90,6 +90,7 @@ jobs:
           export ZEPHYR_TOOLCHAIN_VARIANT=zephyr
           mkdir -p coverage/reports
           pip3 install gcovr==6.0
+          ./scripts/twister -E testplan.json
```

Comment:
this will not work, we will generate this for each platform, so we have later to deal with the same file name. Just pin it tot he platform name and later just pick one to avoid issues. see my workflow on zephyr-testing